iphone/Classes/TiUIiOSStepper.h

```diff
@@ -0,0 +1,18 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2016 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+#ifdef USE_TI_UIIOSSTEPPER
+#import "TiUIView.h"
+
+@interface TiUIiOSStepper : TiUIView {
+@private
+    UIStepper *stepper;
+    UIImage *backgroundImageCache;
```

Comment:
if i don't store the image inside a variable before setting it , it doesn't seem to want to get displayed.  Im following the same logic we do inside the button class.